distributions/distributions.json

```diff
@@ -0,0 +1,11 @@
+{
+    "CumulativeAmountByDistribution": {
+        "2022-07-15": "124734992091552235627767774",
+        "2022-09-01": "183365680150625789032093616",
```

Comment:
What do you think about adding a "total" to the top of the json file? Kind of like the other MerkleDist.json files. For example,

```
{
    "TotalAmount": "340723072250877984478741713",
    "CumulativeAmountByDistribution": {
        "2022-07-15": "124734992091552235627767774",
        "2022-09-01": "183365680150625789032093616",
    ...
```

That way, there's the overall summary total, and then a breakdown of distributions.

Thoughts?

Reply:
Correct. Feel free to rename `TotalAmount` to something more appropriate (eg. `LastCumulativeAmount` or whatever - you have free reign here) if that would make it clearer.

Reply:
Yes, LastCumulativeAmout works for me 👍

Reply:
Done! Could you check the last commit?